### PR TITLE
Mes 7879 unit tests

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Increased coverage

## Checklist
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/store/tests/vehicle-details/cat-adi-part2/__tests__/vehicle-details.cat-adi-part2.reducer.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import {
+  ClearGearboxCategory, DualControlsToggled,
+  GearboxCategoryChanged,
+  MotStatusChanged, SchoolCarToggled,
+  VehicleRegistrationChanged,
+} from '@store/tests/vehicle-details/vehicle-details.actions';
+import { CatADI2UniqueTypes } from '@dvsa/mes-test-schema/categories/ADI2';
+import { vehicleDetailsCatADIPart2Reducer, initialState } from '../vehicle-details.cat-adi-part2.reducer';
+
```
Comment (matthew2564) [inline src/store/tests/vehicle-details/cat-adi-part2/__tests__/vehicle-details.cat-adi-part2.reducer.spec.ts:9]: Can we wrap all these up inside a Describe block please

```describe('vehicleDetailsCatADIPart2Reducer', () => {```

File: src/store/tests/test-summary/cat-a-mod2/__tests__/test-summary.cat-a-mod2.reducer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  DebriefUnWitnessed,
+  DebriefWitnessed,
+  IdentificationUsedChanged,
+  IndependentDrivingTypeChanged,
+  RouteNumberChanged,
+  TrueLikenessToPhotoChanged, WeatherConditionsChanged,
+} from '@store/tests/test-summary/test-summary.actions';
+import { ModeOfTransportChanged } from '@store/tests/test-summary/cat-a-mod2/test-summary.cat-a-mod2.actions';
+import { testSummaryMod2Reducer, initialState } from '../test-summary.cat-a-mod2.reducer';
+
```
Comment (matthew2564) [inline src/store/tests/test-summary/cat-a-mod2/__tests__/test-summary.cat-a-mod2.reducer.spec.ts:13]: Can we wrap all these up inside a Describe block please

```describe('testSummaryMod2Reducer', () => {```

File: src/store/tests/test-summary/cat-a-mod1/__tests__/test-summary.cat-a-mod1.reducer.spec.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  AdditionalInformationChanged,
+  CandidateDescriptionChanged, D255No, D255Yes,
+  DebriefUnWitnessed,
+  DebriefWitnessed,
+  IdentificationUsedChanged,
+  TrueLikenessToPhotoChanged, WeatherConditionsChanged,
+} from '@store/tests/test-summary/test-summary.actions';
+import { CircuitTypeChanged } from '@store/tests/test-summary/cat-a-mod1/test-summary.cat-a-mod1.actions';
+import { testSummaryMod1Reducer, initialState } from '../test-summary.cat-a-mod1.reducer';
+
```
Comment (matthew2564) [inline src/store/tests/test-summary/cat-a-mod1/__tests__/test-summary.cat-a-mod1.reducer.spec.ts:11]: Can we wrap all these up inside a Describe block please

```describe('testSummaryMod1Reducer', () => {```

File: src/store/tests/vehicle-details/cat-adi-part2/__tests__/vehicle-details.cat-adi-part2.reducer.spec.ts (hunk 1 above)
Comment (RLCorp) [reply src/store/tests/vehicle-details/cat-adi-part2/__tests__/vehicle-details.cat-adi-part2.reducer.spec.ts:9]: adressed

File: src/store/tests/test-summary/cat-a-mod2/__tests__/test-summary.cat-a-mod2.reducer.spec.ts (hunk 2 above)
Comment (RLCorp) [reply src/store/tests/test-summary/cat-a-mod2/__tests__/test-summary.cat-a-mod2.reducer.spec.ts:13]: adressed

File: src/store/tests/test-summary/cat-a-mod1/__tests__/test-summary.cat-a-mod1.reducer.spec.ts (hunk 3 above)
Comment (RLCorp) [reply src/store/tests/test-summary/cat-a-mod1/__tests__/test-summary.cat-a-mod1.reducer.spec.ts:11]: adressed

**Review by matthew2564 — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**